pub-critic/src/app/page.module.scss

```diff
@@ -0,0 +1,96 @@
+.Container {
+    padding: 70px 5vw;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    background-color: #151515;
+    .Page {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+        .Image {
+            width: 100vw;
+            height: 100vh;
+            position: relative;
+            > img {
+                width: 100%;
+                height: 100%;
+                object-fit: cover;
+            }
+            .Backdrop {
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
+                background-color: black;
+                opacity: 0.5;
+            }
+            .Title {
+                position: absolute;
+                top: 50;
+                left: 0;
+                width: 100%;
+                height: 100%;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                align-items: center;
+                color: white;
+                font-size: 2vw;
+                font-weight: 500;
+                > h1 {
+                    font-size: 4vw;
+                    font-weight: 900;
+                    margin-bottom: 50px;
+                    @media (max-width: 768px) {
+                        font-size: 8vw;
+                    }
+                }
+                > span {
+                    @media (max-width: 768px) {
+                        font-size: 3vw;
+                    }
+                    font-size: 2vw;
+                    font-weight: 500;
+                }
+            }
+        }
+        .Section {
+            width: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: center;
+            margin-top: 40px;
+            @media (max-width: 768px) {
+                align-items: center;
+            }
+            > h1 {
+                font-size: 4vw;
+                font-weight: 900;
+                margin-bottom: 20px;
+                @media (max-width: 768px) {
+                    font-size: 8vw;
+                }
+            }
+            .NotFound {
+                font-size: 2.5vw;
+                font-weight: 500;
+                color: white;
+                width: 100%;
+                height: 500px; // approximate height of game component
```

Comment:
not responsive on larger screens...use vw or %

Reply:
:white_check_mark: Should I also make height for the game component responsive, and is it smart to connect height to vw instead of vh?

Reply:
Remind me to show you some examples tomorrow